_docs/guides/integrating-vms.md

```diff
@@ -12,16 +12,16 @@ This sample deploys the Bookinfo services across Kubernetes and a set of
 Virtual Machines, and illustrates how to use Istio service mesh to control
 this infrastructure as a single mesh.
 
+> Note: this guide is still under development and only tested on GCE.
+
```

Comment:
GCE -> Google Cloud Platform.

Reply:
May be even add something like on IBM Bluemix or most other platforms where overlay network of Pods is isolated from VM network, VMs cannot initiate any direct communication to Kubernetes Pods when using Istio.